test/remote/gateways/remote_versa_pay_test.rb

```diff
@@ -0,0 +1,188 @@
+require 'test_helper'
+
+class RemoteVersaPayTest < Test::Unit::TestCase
+  def setup
+    @gateway = VersaPayGateway.new(fixtures(:versa_pay))
+    @bad_gateway = VersaPayGateway.new(api_token: 'bad_token', api_key: 'bad_key')
+
+    @amount = 500
+    @credit_card = credit_card('4895281000000006', verification_value: '123', month: 12, year: Time.now.year + 1)
+    @credit_card_match_cvv = credit_card('4895281000000006', verification_value: '234', month: 12, year: Time.now.year + 1)
+    @credit_card_not_match_cvv = credit_card('4895281000000006', verification_value: '345', month: 12, year: Time.now.year + 1)
+    @decline_credit_card = credit_card('4264280001234500')
+    @no_valid_date_credit_card = credit_card('4895281000000006', month: 9, year: Time.now.year - 1)
+    #     @master_credit_card = credit_card('5399999999999999')
```

Comment:
💬 Comment:

Pointing at this in case is not something that you have saved for the future.